### ISSUE TITLE
Questions about vanilla OPD implementation details

### ISSUE THREAD

**Issue by wuycM** [tiankanghui/ViCuR#4]
Hi, thank you for releasing this interesting work!

I have a few questions about the vanilla OPD experiments:

1. During training on Vision-R1, did you add the following instruction to the student rollout prompt and teacher input?
instruction_following = (
    r"You FIRST think about the reasoning process as an internal monologue and then provide the final answer. "
    r"The reasoning process MUST BE enclosed within <think> </think> tags. "
    r"The final answer MUST BE put in \boxed{}."
)
2. How was the reported checkpoint selected? Was it the checkpoint with the highest validation accuracy, or simply the final checkpoint after training?
3. Could you share the configuration file or training script for the vanilla OPD experiments?

Thank you so much for your time!

**Comment by tiankanghui** [comment]
Apologies for the delayed response, and thanks for your interest in our work! To answer your questions: yes, we did add that instruction, which you can verify directly in our provided data files. For the checkpoint, we simply used the final one after training. As for the configs, the exact settings are already available in the training scripts in this repo and detailed in the paper's appendix. Feel free to reach out if you need any further clarification!
